yt_dlp/extractor/roya.py

```diff
@@ -0,0 +1,61 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError
+
+
+class RoyaIE(InfoExtractor):
```

Comment:
Let's name this `RoyaLiveIE`, as `RoyaIE` would be the extractor for vods
```suggestion
class RoyaLiveIE(InfoExtractor):
```